hw14/src/test/java/Hw14.java

```diff
@@ -0,0 +1,77 @@
+import org.openqa.selenium.Beta;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.awt.*;
+
+public class Hw14 {
+    private WebDriver driver;
+    public static final String LOG = "zhenyamoroz2208";
+    public static final String PAS = "0000adab";
+
+    @BeforeMethod
+    public void setUp() {
+        driver = new ChromeDriver(); //Открываем браузер
+        driver.manage().window().maximize(); //Открываем окно браузера на весь экран
+        driver.get("https://www.i.ua/"); //Переходим по URL
+    }
+     @AfterMethod
+       public void tearDown() {
+
+         driver.close(); //Закрываем браузер
+          driver.quit(); //Закрываем браузер
+     }
+    @Test
+    public void validLog()  {
+        driver.findElement(By.linkText("Вход")).click();
+        driver.findElement(By.cssSelector("fieldset input[name='login']")).sendKeys(LOG);
+        driver.findElement(By.cssSelector("fieldset input[name='pass']")).sendKeys(PAS);
+        driver.findElement(By.cssSelector("input:nth-child(7)")).click();
+
+
+    }
+    @Test
+    public void ivalidLog()  {
+        driver.findElement(By.linkText("Вход")).click();
+        driver.findElement(By.cssSelector("fieldset input[name='login']")).sendKeys("");
+        driver.findElement(By.cssSelector("fieldset input[name='pass']")).sendKeys("");
+        driver.findElement(By.cssSelector("input:nth-child(7)")).click();
+        WebElement searchResult = driver.findElement(By.cssSelector(".right a[title='выйти']")); //Получаем результат поиска

```

Comment:
Comment does not match with code
searchResult variable name does not match with the meaning